common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -5,17 +5,17 @@
 @sectionList(topLevelSection: NewNavigation.TopLevelSection) = {
     <li class="main-navigation__item navigation-border">
             <!-- TODO: Firefox -->
-        <details class="js-close-nav-list main-navigation__primary-list" id="primary-list-@topLevelSection.name">
+        <details class="js-close-nav-list main-navigation__primary-list" id="primary-list-@topLevelSection.name" open>
```

Comment:
Note: `open` is added to all the details elements so they are all open by default